modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -14939,19 +14951,22 @@ union TargetIntegrationTimeOutcome = TargetIntegrationTime | TargetError
 # ITC results for a target, can be a single result or one per CCD
 type TargetIntegrationTime {
   # All CCD results for the target
-  all: [IntegrationTime!]!
+  all: [IntegrationTime!]
```

Comment:
This change induced quite a bit of changes in the server.
if you call with time and count mode you don't get `all`, nor `index` nor `selected`. 
This could be emulated by using the time and count passed as a param but it seems no correct to do so

Reply:
Could you explain a bit more why the mode would have an impact on these fields?  I think as a user, I don't care much about each CCD anyway and just want the time, count, and s/n regardless of what the mode was.  How do I get those values?

Reply:
you provide them in timeandcount mode, they will be returned in the `exposureMode` too

Reply:
```graphql
      query Spectroscopy($spec: SpectroscopyInput!) {
        spectroscopy(input: $spec) {
          exposureTimeMode {
            timeAndCount {
              time {
                seconds
              }
              count
            }
          }
          targetTimes {
            ... on TargetIntegrationTime {
              signalToNoiseAt {
                wavelength {
                  nanometers
                }
                single
                final
              }
            }
          }
        }
        
      }
```

Reply:
I see your point though. it seems odd to me to return the exposure time requested as the result but it would make queries easier. Let's do that

Reply:
I reverted the schema change, now either exposure mode could be handle by a the same query

```graphql